spec/windows.dd

```diff
@@ -0,0 +1,46 @@
+Ddoc
+
+$(SPEC_S Windows Programming,
+
+$(HEADERNAV_TOC)
+
+$(P This covers Windows programming with 32 bit OMF, 32 bit Mscoff,
+64 bit Mscoff, console programs, GUI programs, and DLLs.
+)
+
+$(H2 $(LNAME2 mscoff, Windows 32 and 64 bit MSCOFF Programs))
+
+    $(P 32 bit and 64 bit MSCOFF programs use the Microsoft Visual C/C++ compiler as the Associated
+    C Compiler, generate object files in the MSCOFF format, and use the Microsoft linker
+    to link them.
+    )
```

Comment:
That'll need to cover mingw and lld (which you can't enable if you have VS).

Reply:
I don't know the ecosystem around mingw/lld. Someone who does will have to write it.

Reply:
> 32 bit and 64 bit MSCOFF programs use the Microsoft Visual C/C++ compiler as the Associated C Compiler, generate object files in the MSCOFF format, and use the Microsoft linker if available to link them. If not available mingw import libraries and LLD linker will be used instead.

Shouldn't need much, like what I wrote above.

Reply:
That should specify that it's the MinGW import libraries and LLD that ship with dmd. Anyone attempting to use the import libraries in the MinGW distro may encounter difficulties, so it's important to clarify that.